make_timeline.py

```diff
@@ -69,7 +69,7 @@ def build(self):
 		self.g_axis.translate(0, y_axis)
 		self.drawing.add(self.g_axis)
 		# finally set the height on the drawing
-		self.drawing['height'] = height
+		self.drawing['height'] = height * 1.4
```

Comment:
Hi Jason,

Thanks for the tool!

The width I had to adjust it because of an outlier, eg. Apr 01,....long
time went by... And then several events in July and August. Adjusting the
width was a dirty hack not to have to set the start date to eg. Jan 01. The
label of the event in Apr was also very lengthy.

Btw, there other cases that I had to make adjustments manually on the
generated svg, eg in case the timeline was 4 months long, and I had several
events happening with only minutes apart each other's. IMHO it would be
could to have some kind of "overlapping detection" which would then
"artificially" separate x coordinates of those neighboring events and
indicate it visually somehow... Let me know if you need the json file with
the example.

Btw, thanks for the great tool. Helps me save a lot of time.

Regards
Joao




On Saturday, 13 August 2016, Jason Reisman <notifications@github.com> wrote:

> In make_timeline.py
> <https://github.com/jasonreisman/Timeline/pull/2#discussion_r74687013>:
>
> > @@ -69,7 +69,7 @@ def build(self):
> >  		self.g_axis.translate(0, y_axis)
> >  		self.drawing.add(self.g_axis)
> >  		# finally set the height on the drawing
> > -		self.drawing['height'] = height
> > +		self.drawing['height'] = height * 1.4
>
> Just out of curiosity, why did you want to increase the image height by
> 40%? Rather than hardcoding the number, maybe this could be a command line
> argument?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jasonreisman/Timeline/pull/2/files/0b304779860400d9d6dfb50605f7fc38fec2303f#r74687013>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC5nWSsF59Jv12bKCTnPSdMzdlyCLfEBks5qfccEgaJpZM4Ji6M3>
> .
>